htdocs/core/class/html.formfile.class.php

```diff
@@ -628,9 +628,17 @@ public function showdocuments($modulepart, $modulesubdir, $filedir, $urlsource,
 				// For normalized external modules.
 				else {
 				    $file = dol_buildpath('/'.$modulepart.'/core/modules/'.$modulepart.'/modules_'.$submodulepart.'.php', 0);
-					$res = include_once $file;
+					if (file_exists($file))
```

Comment:
Why do you need this.
With the 
$file = dol_buildpath('/'.$modulepart.'/core/modules/'.$modulepart.'/modules_'.$submodulepart.'.php', 0);
$res = include_once $file;

You can already include the class ModulePDFyourmodulename that can hae a method liste_modeles() to return list of document templates.